java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/SSL/TrustManager.java

```diff
@@ -29,15 +29,13 @@ class TrustManager {
             parse(properties.getProperty(key), _rejectClient, _acceptClient);
             key = "IceSSL.TrustOnly.Server";
             parse(properties.getProperty(key), _rejectAllServer, _acceptAllServer);
-            Map<String, String> dict =
-                properties.getPropertiesForPrefix("IceSSL.TrustOnly.Server.");
+            Map<String, String> dict = properties.getPropertiesForPrefix("IceSSL.TrustOnly.Server.");
+
             for (Map.Entry<String, String> p : dict.entrySet()) {
                 key = p.getKey();
                 String name = key.substring("IceSSL.TrustOnly.Server.".length());
-                List<List<RFC2253.RDNPair>> reject =
-                    new LinkedList<List<RFC2253.RDNPair>>();
-                List<List<RFC2253.RDNPair>> accept =
-                    new LinkedList<List<RFC2253.RDNPair>>();
+                List<List<RFC2253.RDNPair>> reject = new LinkedList<>();
+                List<List<RFC2253.RDNPair>> accept = new LinkedList<>();
```

Comment:
Java let's you omit generic types when constructing new instances.
I removed all of the unnecessary generics from this file, so that lots of stuff would fit on a single line.